code/game/objects/structures/fireaxe.dm

```diff
@@ -1,6 +1,6 @@
 /obj/structure/fireaxecabinet
 	name = "fire axe cabinet"
-	desc = "There is a small label that reads \"For Emergency use only\" along with details for safe use of the axe. As if."
+	desc = "There is a small label that reads \"For Emergency use only\" along with details for safe use of the axe. As if. Alt-click to open and close."
```

Comment:
this isn't quite what we were looking for. look at this for an example.
https://github.com/tgstation/tgstation/blob/0b03db27bff11b2276ed0eb879360baa8bb96512/code/game/objects/items/devices/radio/intercom.dm#L28-L33